speechtool.py

```diff
@@ -28,11 +33,14 @@
 
 from . import *
 
+from ..core.managers import edit_delete, edit_or_reply
+from . import deEmojify, reply_id
```

Comment:
why?

Reply:
It will not make .tts in unknown format, Sometimes, we get voice delivered in unknown format as voice.ogg, its recognised by telegram as unknown format, so it will fix the issue.

Reply:
still, why the imports?

Reply:
Devesh Said If Want to contribute then Raise pill request, I also wanna be in team, I asked him...